launcher/ui/pages/instance/ModFolderPage.cpp

```diff
@@ -181,6 +181,7 @@ void ModFolderPage::downloadDialogFinished(int result)
         } else {
             qWarning() << "ResourceDownloadDialog vanished before we could collect tasks!";
         }
+        m_downloadDialog->deleteLater();
```

Comment:
You may have a crash here: previously you checked if the pointer still exist but here you just use it.

Reply:
moved it to the end of the function with it's own check